docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+```
+CORE_PEER_LOCALMSPID="Org2MSP"
+CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+export CORE_PEER_ADDRESS=localhost:9051
+```
+
+Issue the following command to install the chaincode:
+```
+peer lifecycle chaincode install fabcar.tar.gz
+```
+
+The chaincode is built by the peer when the chaicode is installed. The install command will return any build errors from the chaincode if there is a problem with the smart contract code.
+
+## Approve a chaincode definition
+
+After you install the chaincode package, you need to approve a chaincode definition for your organization. The definition includes the important parameters of chaincode governance such as the name, version, and the chaincode endorsement policy.
+
+The set of channel members who need to approve a chaincode before it can be deployed is governed by the `Application/Channel/lifeycleEndorsement` policy. By default, this policy requires that a majority of channel members need to approve a chaincode before it can used on a channel. Because we have only two organizations on the channel, and a majority of 2 is 2, we need approve a chaincode definition of Fabcar as Org1 and Org2.
+
+If an organization installed the chaincode, they need to include the packageID in the chaincode definition approved by their organization. The package ID is used to associate the chaincode installed on a peer with an approved chaincode definition. You can find the package ID of a chaincode by using the `peer lifecycle chaincode queryinstalled` command to query your peer.
+```
+peer lifecycle chaincode queryinstalled
+```
+
+The package ID is the combination of the chaincode label and a hash of the chaincode bytes. You should see output similar to the following:
+```
+Get installed chaincodes on peer:
+Package ID: fabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3, Label: fabcar_1
+```
+
+We are going to use the package ID when we approve the chaincode, so let's go ahead and save it as an environment variable. Paste the package ID returned by `peer lifecycle chaincode queryinstalled` into the command below. The package ID may not be the same for all users, so you need to complete this step using the package ID returned from your command window.
+```
+CC_PACKAGE_ID=fabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3
+```
+
+Because you set the environment variables to operate the `peer` CLI as the Org2 admin, we can approve the chaincode definition of Fabcar for Org2. Chaincode is approved at the organization level, so the command only needs to target one peer. The approval is distributed to peers within each organization using gossip.
```

Comment:
The approval is distributed to peers within each organization using gossip.
or
The approval is distributed to the other peers within the organization using gossip.
?

Reply:
done.